themes/bootstrap3/js/config.js

```diff
@@ -0,0 +1,21 @@
+/*global VuFind */
+VuFind.register('config', function Config() {
+  var _config = {};
+
+  function add(config) {
+    for (var i in config) {
+      if (Object.prototype.hasOwnProperty.call(config, i)) {
+        _config[i] = config[i];
+      }
+    }
+  }
+
+  function get(key) {
```

Comment:
Might we want a mechanism for default values if keys are unset, or is the expectation that we'll deal with all that externally?

Reply:
I would handle this externally like "VuFind.config.get('config')?.Site?.title ?? 'Default Title'".